content/reference/templates/_index.md

```diff
@@ -45,8 +45,8 @@ Templating system uses standard ASCII quotation marks:\
 0x22 > `"` for straight double quotes, 0x27 > `'`for apostrophes and 0x60 `` ` `` for backticks/back quotes; so make
 sure no "smart-quotes" are being used.
 
-The difference between back quotes and double quotes in string literals is covered
-[here](https://go.dev/ref/spec#String\_literals).
+The difference between back quotes and double quotes in string literals is covered in
+[Golang reference documentation](https://go.dev/ref/spec#String\_literals).
```

Comment:
```suggestion
The difference between back quotes and double quotes in string literals is covered in
the [Go Language Specification](https://go.dev/ref/spec#String\_literals).
```

The name of this popular programming language is Go, the name of the linked document is its language specification.
We have more instances of "Golang" instead of "Go"; please change those as well.